spec/g10_certification_test_kit/bulk_data_group_export_spec.rb

```diff
@@ -192,17 +193,17 @@ def run(runnable, inputs = {})
       expect(result.result_message).to eq('Server response did not have Content-Location in header')
     end
 
-    # NOTE: Test commented out as it takes three minutes to fail
-    # it 'skips when server only returns "202 Accepted", and not "200 OK" in the allowed timeframe' do
-    #   stub_request(:get, "#{polling_url}")
-    #     .with(headers: { 'Authorization' => "Bearer #{bearer_token}" } )
-    #     .to_return(status: 202)
+    it 'skips when server only returns "202 Accepted", and not "200 OK" in the allowed timeframe' do
+      stub_request(:get, polling_url.to_s)
+        .with(headers: { 'Authorization' => "Bearer #{bearer_token}" })
+        .to_return(status: 202)
 
-    #   result = run(runnable, base_input)
+      allow_any_instance_of(runnable).to receive(:sleep) { |_o, time| Timecop.travel(Time.now + time) }
```

Comment:
If you're just mocking `sleep`, you don't need timecop. You can just make `sleep` do nothing.